internal/test/e2e/gouging_test.go

```diff
@@ -91,4 +95,19 @@ func TestGouging(t *testing.T) {
 	if err := w.DownloadObject(context.Background(), &buffer, api.DefaultBucketName, path, api.DownloadObjectOptions{}); err == nil {
 		t.Fatal("expected download to fail", err)
 	}
+
+	// try optimising gouging settings
+	resp, err := cluster.Autopilot.EvaluateConfig(context.Background(), test.AutopilotConfig, gs, test.RedundancySettings)
+	tt.OK(err)
+	if resp.Recommendation == nil {
+		t.Fatal("expected recommendation")
+	}
+
+	// set optimised settings
+	tt.OK(b.UpdateSetting(context.Background(), api.SettingGouging, resp.Recommendation.GougingSettings))
+
+	// renter should recover and be able to upload again
+
+	// upload some data - should fail
```

Comment:
The comment above indicates that uploads should recover and yet we expect it to fail here. Even if we do expect it to fail, we should add a test care where we apply a recommendation which makes uploads (or downloads) succeed where they failed before.